javascript-data-model/order-history.js

```diff
@@ -43,16 +43,19 @@ var orderHistory = [
     orderPlaced: 'July 3, 2020',
     orderTotalPrice: '$138.93',
     expectedDelivery: 'July 5, 2020',
-    item1Description: {
-      category: 'electronics',
-      itemName: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
-      priceBeforeTax: '$94.95'
-    },
-    item2Description: {
-      category: 'book',
-      itemName: 'The Art of Sql',
-      priceBeforeTax: '$33.99'
-    },
+    orderDescription:
+    [
+      {
+        category: 'electronics',
+        itemName: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+        priceBeforeTax: '$94.95'
+      },
+      {
+        category: 'book',
+        itemName: 'The Art of Sql',
+        priceBeforeTax: '$33.99'
+      }
+    ],
```

Comment:
For this order, the value of this property is an array, but in your other orders, it is an object. This is an inconsistent format, and will lead to bugs.